src/main/java/io/aiven/kafka/connect/http/config/HttpSinkConfig.java

```diff
@@ -406,7 +408,7 @@ private static void addTimeoutConfigGroup(final ConfigDef configDef) {
             ConfigDef.Importance.LOW,
             "HTTP Response timeout (seconds). Default is 30 seconds.",
             TIMEOUT_GROUP,
-            groupCounter++,
+            1,
```

Comment:
This counter looks cumbersome here, but it's uniform with the other groups (where there's more than one configuration). Maybe keep it as is?

Reply:
I can add the Spotbugs suppression.